src/wasm/api.rs

```diff
@@ -0,0 +1,113 @@
+use std::collections::HashMap;
+
+use wasm_bindgen::prelude::*;
+
+use crate::allocator::{Allocator, NodePtr};
+use crate::cost::Cost;
+use crate::err_utils::err;
+use crate::f_table::{f_lookup_for_hashmap, FLookup};
+use crate::more_ops::op_unknown;
+use crate::node::Node;
+use crate::reduction::Response;
+use crate::run_program::{run_program, OperatorHandler};
+use crate::serialize::{node_from_bytes, node_to_bytes};
+
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
+// allocator.
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+struct OperatorHandlerWithMode {
+    f_lookup: FLookup,
+    strict: bool,
+}
+
+impl OperatorHandler for OperatorHandlerWithMode {
+    fn op(
+        &self,
+        allocator: &mut Allocator,
+        o: NodePtr,
+        argument_list: NodePtr,
+        max_cost: Cost,
+    ) -> Response {
+        let b = &allocator.atom(o);
+        if b.len() == 1 {
+            if let Some(f) = self.f_lookup[b[0] as usize] {
+                return f(allocator, argument_list, max_cost);
+            }
+        }
+        if self.strict {
+            err(o, "unimplemented operator")
+        } else {
+            op_unknown(allocator, o, argument_list, max_cost)
+        }
+    }
+}
+
+#[wasm_bindgen]
+pub fn run_clvm(program: &[u8], args: &[u8]) -> Vec<u8> {
+    let quote_kw: u8 = 1;
+    let apply_kw: u8 = 2;
+    let max_cost: Cost = 1_000_000_000_000_000;
```

Comment:
this is very high. you don't think we should set this to at most the max block cost?

Reply:
(I agree. See above.)